modoboa/core/commands/templates/settings.py.tpl

```diff
@@ -114,15 +114,15 @@ MIDDLEWARE = (
 )
 
 AUTHENTICATION_BACKENDS = (
-    # 'modoboa.lib.authbackends.LDAPBackend',
-    # 'modoboa.lib.authbackends.SMTPBackend',
+    #'modoboa.lib.authbackends.LDAPBackend',
+    #'modoboa.lib.authbackends.SMTPBackend',
     'django.contrib.auth.backends.ModelBackend',
 )
 
 # SMTP authentication
-# AUTH_SMTP_SERVER_ADDRESS = 'localhost'
-# AUTH_SMTP_SERVER_PORT = 25
-# AUTH_SMTP_SECURED_MODE = None  # 'ssl' or 'starttls' are accepted
+#AUTH_SMTP_SERVER_ADDRESS = 'localhost'
+#AUTH_SMTP_SERVER_PORT = 25
+#AUTH_SMTP_SECURED_MODE = None  # 'ssl' or 'starttls' are accepted
```

Comment:
pep8 says we need a space after a hash comment.

Reply:
It was to distinguish between comments and variables that can be decommented. I can either keep a space or add an exception on this file for this rule to make pep8 happy. What do you prefer?

Reply:
Ok I see. Let's add a exception.

Reply:
I have excluded this error only for this file in the flake8 configuration, is it enough? pep8 seems to be unmaintained for a while, and I didn't find which linter should be run and where.